lib/jsonApi.js

```diff
@@ -1,6 +1,7 @@
 /* @flow weak */
 "use strict";
 var jsonApi = module.exports = { };
+jsonApi._version = require(require("path").join(__dirname, "../package.json")).version;
```

Comment:
It would definitely be nicer, but unfortunately I need this the be the required package version when I require it in the data stores. :disappointed: `process.env.npm_package_version` would give me the main package's version.